README.md

```diff
@@ -13,6 +13,7 @@ This repository contains an easy to replicate recipe to spin up a [Kopano](https
 - now run `docker-compose up` and you will see how the remaining Docker images are pulled and then everything is started
 - after startup has succeeded you can access the Kopano WebApp by going to `https://kopano.demo/webapp`
 - you can also access phpLDAPadmin by going to `https://kopano.demo/ldap-admin`
+  - you can access the ldap admin with: `dc=admin,dn=kopano,dn=demo` as login dn and the `LDAP_ADMIN_PASSWORD` environment variable provided by the .env file (which is generated by the setup.sh)
```

Comment:
The username is also dynamic. Maybe also tell users to read it from the .env file?

Reply:
I did not find the ldap admin username in the env. file. I did also not find it in the  ldap-admin environment setup of the compose file. So i luckily estimated the username based from other setups. Are you sure, that the ldap-admin username is dynamic?

Reply:
Yes, you're right. It is indeed not totally clear just from looking at compose or env. The first part (so `dc=admin,` is given), but the rest is just your ${LDAP_BASE_DN}

Reply:
Now the readme is more specific about reading the login information from the .env file

Reply:
ah, I see I mixed up it up here as well, its cn=admin. I'll quickly fix this up and then merge this pr.

Thanks for your work!